registration_ref/sota_toml.py

```diff
@@ -82,6 +83,7 @@ def sota_toml_fmt(overrides=None, sota_config_dir="/var/sota"):
                 v = v.format(
                     gateway_server=Settings.DEVICE_GATEWAY_SERVER,
                     sota_config_dir=sota_config_dir,
+                    hardware_id=hardware_id,
```

Comment:
We don't really need this anymore. The lmp now always includes /usr/lib/sota/conf.d/40-hardware-id.toml. If a user *really* wants to override the hardware-id they could use the existing `overrides` parameter.

I guess we should probably update lmp-device-register to do it this way so that we can drop this logic in our backend also.

Reply:
I run aklite on non LmP-based machine (locally, server, etc), that's why I hit it, probably we will need it in the future for automated tests.

Reply:
how about we drop this here and add a bug to lmp-device-register for setting hwid as an "overrides" parameter. then we can remove a couple of lines of code from the ota-lite code base?

Reply:
ok